cmd/ion-go/eventwriter.go

```diff
@@ -0,0 +1,268 @@
+package main
```

Comment:
I don't find individual file headers necessary or useful, but if you and/or your legal team do I'm willing to add them. Fair warning, if I do they'll say "Copyright David Murray" rather than "Copyright Amazon." :)

Reply:
Legal confirmed it is optional. I'd rather we skip them :)